python_modules/dagster/dagster/components/resolved/context.py

```diff
@@ -18,8 +18,17 @@
 T = TypeVar("T")
 
 
-def env_scope(key: str) -> Optional[str]:
-    return os.environ.get(key)
+class EnvScope:
+    def __call__(self, key: str) -> Optional[str]:
+        return os.environ.get(key)
+
+    def __getattr__(self, key: str) -> Optional[str]:
+        if key.startswith("jinja"):
+            raise AttributeError(f"{key} not found")
```

Comment:
whats this about ?

Reply:
jinja does an attr check as part of applying a UDF - added a comment:
```
../../.pyenv/versions/3.11.9/envs/dagster/lib/python3.11/site-packages/jinja2/utils.py:92: in from_obj
    if hasattr(obj, "jinja_pass_arg"):
```